serverless/question-service/functions/index.ts

```diff
@@ -0,0 +1,56 @@
+import * as functions from "firebase-functions";
+
+import { updateQuestionsContent as contentEntry } from "./logic/questions-content.main";
+import { updateQuestionsSummary as summaryEntry } from "./logic/questions-summary.main";
+
+export const updateQuestionsContent = functions
+  .region("asia-southeast1")
+  .runWith({
+    timeoutSeconds: 540,
+    memory: "256MB",
+    secrets: ["DATABASE_URL"],
+  })
+  .pubsub.schedule("10 */1 * * *")
+  .onRun(async () => {
+    try {
+      await contentEntry(process.env.DATABASE_URL ?? "NOT FOUND");
+    } catch (error) {
+      functions.logger.error(error);
+    }
+  });
+
+export const updateQuestionsSummary = functions
+  .region("asia-southeast1")
+  .runWith({
+    timeoutSeconds: 540,
+    memory: "256MB",
+    secrets: ["DATABASE_URL"],
+  })
+  .pubsub.schedule("0 */1 * * *")
```

Comment:
is this scheduled for once a day?

Reply:
@aidanaden this is by the hour -- I don't know when LC updates their daily question but this we would at worst be 1hr behind them.